main.go

```diff
@@ -18,9 +18,16 @@ const LOGGER_TYPE_LOG = "log"
 const LOGGER_TYPE_ERROR = "error"
 
 type Context struct {
-	Logger Logger
-	Req    Request
-	Res    Response
+	logger Logger
```

Comment:
why was this made lowercase? From the examples Ive seen online, you either use all lowercase field names or all uppercase field names within a structure.. We dont use a mix of both.

Reply:
`lowercase` is private - only methods on struct can access it.
`Uppercase` is public - anyone with access to struct can read it.

This was made lowercase on purpose, to hide it from developer. Because IDE was auto-suggesting `Context.Logger()` before suggesting `Context.Log()`. And you are not meant to use Logger directly. So we hid it.